app/models/katello/content_view_package_group_filter_rule.rb

```diff
@@ -7,6 +7,14 @@ class ContentViewPackageGroupFilterRule < Katello::Model
                :inverse_of => :package_group_rules,
                :foreign_key => :content_view_filter_id
 
+    # rubocop:disable Rails/HasManyOrHasOneDependent
+    has_one :package_group,
+            class_name: "Katello::PackageGroup",
+            inverse_of: :content_view_filter_rules,
+            primary_key: :uuid,
+            foreign_key: :pulp_id
+    # rubocop:enable Rails/HasManyOrHasOneDependent
```

Comment:
i think you want to use belongs_to here, thats what the rubocop was complaining about.   I think all of the options are the same and it works very similarly.